doc/gendocs.sh

```diff
@@ -3,5 +3,11 @@ set -xeuo pipefail
 rm -rf _build
 rm -rf apidoc
 sphinx-apidoc -T -e -o apidoc ../src/commissaire/
-# TODO: Add the other repos here to get full api docs
+if [ -d "../../commissaire-service/" ]; then
+  sphinx-apidoc -T -e -o apidoc ../../commissaire-service/src/commissaire_service/
+fi
+if [ -d "../../commissaire-http/" ]; then
```

Comment:
If I read this right, this is actually looking *outside* the git repo and making assumptions about the local development environment.  I'd suggest a comment to that effect:
```
This requires the service and http repositories to be cloned in a common parent directory,
and the repository directories must be named "commissaire-service" and "commissaire-http".
```